pkgs/top-level/python-packages.nix

```diff
@@ -1191,7 +1191,13 @@ self: super: with self; {
 
   paddle2onnx = callPackage ../development/python-modules/paddle2onnx { };
 
-  paddlepaddle = callPackage ../development/python-modules/paddlepaddle { };
+  paddlepaddle = callPackage ../development/python-modules/paddlepaddle {
+    cudaSupport = pkgs.config.cudaSupport or false;
+  };
+
+  paddlepaddle-gpu = callPackage ../development/python-modules/paddlepaddle {
+    cudaSupport = true;
+  };
```

Comment:
I don't think we need to add the gpu version to the top level since it is properly available through nixpkgs.config.